test/integration/server/helpers/helpers.go

```diff
@@ -94,40 +97,50 @@ func NewGithubClient(ctx context.Context, token string) *ghAPI.Client {
 	return ghAPI.NewClient(tc)
 }
 
-const InAppRoot = ".wego"
-const ExternalConfigRoot = ""
+const InAppRoot = ".weave-gitops"
```

Comment:
Thoughts on exporting `automationRoot` from the automation package instead of defining it again here?

Reply:
The idea here was to use a separate set of definitions so as not to just hide a bug by using the same calls that might contain the bug